io.gitlab.sulincix.sterm.yaml

```diff
@@ -0,0 +1,66 @@
+app-id: io.gitlab.sulincix.sulincix
```

Comment:
the repo name is `sterm`. so this doesn't match

Reply:
and it doesn't match the filename of this file either (which seems correct based on the comment above)